config/resources/lpdc.lisp

```diff
@@ -0,0 +1,58 @@
+(define-resource abstract-public-service ()
+  :class (s-prefix "lpdcExt:AbstractPublicService")
+  :properties `((:name :language-string-set ,(s-prefix "dct:title"))
+                (:description :language-string-set ,(s-prefix "dct:description"))
+                (:additional-description :language-string-set ,(s-prefix "lpdcExt:additionalDescription"))
+                (:keyword :language-string-set ,(s-prefix "dcat:keyword"))
+                (:regulations :language-string-set ,(s-prefix "lpdcExt:regulation"))
+                (:exceptions :language-string-set ,(s-prefix "lpdcExt:exception"))
+                (:start-date :datetime ,(s-prefix "schema:startDate"))
+                (:end-date :datetime ,(s-prefix "schema:endDate"))
+                (:created :datetime ,(s-prefix "dct:created"))
+                (:modified :datetime ,(s-prefix "dct:modified"))
+                (:product-id :string ,(s-prefix "schema:productID")))
```

Comment:
I was thinking of having consistent spacing in our migration and resource code in general. For example,
```
((:name                   :language-string-set  ,(s-prefix "dct:title"))
 (:description            :language-string-set  ,(s-prefix "dct:description"))
 (:additional-description :language-string-set  ,(s-prefix "lpdcExt:additionalDescription"))
 (:keyword                :language-string-set  ,(s-prefix "dcat:keyword"))
 (:regulations            :language-string-set  ,(s-prefix "lpdcExt:regulation"))
 (:exceptions             :language-string-set  ,(s-prefix "lpdcExt:exception"))
 (:start-date             :datetime             ,(s-prefix "schema:startDate"))
 (:end-date               :datetime             ,(s-prefix "schema:endDate"))
 (:created                :datetime             ,(s-prefix "dct:created"))
 (:modified               :datetime             ,(s-prefix "dct:modified"))
 (:product-id             :string               ,(s-prefix "schema:productID")))
```

Given that it's data defined in Lisp code, I find it visually better to have spacing in order to notice the separation better. Do you think it would look better this way?

Reply:
agree with @Windvis